tests/p_centos-release/centos-release_centos_gpg.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/sh
 # Author: Athmane Madjoudj <athmanem@gmail.com>
 
-t_Log "Running $0 - CentOS RPM GPG Keys exist."
+t_Log "Running $0 - $os_name RPM GPG Keys exist."
 
-file /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS*  >/dev/null 2>&1 && \
-file /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Security*  >/dev/null 2>&1
+if [[ $is_almalinux == "yes" ]]; then
```

Comment:
Can we use $os_name here instead of hard-coding the paths?

Reply:
Ok, I will try to use this way